base/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM azul/zulu-openjdk-alpine:8
```

Comment:
how are we choosing this? it seems to differ from the base image in cp-docker-images

Reply:
Alpine's a popular choice for minified images. Since we don't actually need Debian for anything, I thought I'd try to slim things down.

Reply:
Sure, but keep in mind compatibility concerns. We may choose to do something incompatible, but I think this means even the system package tooling will completely change, right?

Reply:
The system packages are indeed totally different in Alpine. There are probably use cases unknown to me where this could be disruptive. I can give this some more thought and research.